addons/l10n_jo_edi/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_jo_edi_client_id = fields.Char("Jordan EINV Client ID")
```

Comment:
put the keywords when creation fields
```suggestion
    l10n_jo_edi_client_id = fields.Char(string="Jordan EINV Client ID")
```